test/unit/reduce_active_set.c

```diff
@@ -58,7 +58,7 @@ int main(void)
 
     for (i = 0; i < SHMEM_REDUCE_SYNC_SIZE; i++) {
         max_psync[i] = SHMEM_SYNC_VALUE;
-        max_psync[i] = SHMEM_SYNC_VALUE;
+        mix_psync[i] = SHMEM_SYNC_VALUE;
```

Comment:
Whoops, I think you meant `min_psync`.

Reply:
Doh!  Yep, should be `min_psync`.  updated PR.